dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-pingidentity/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "backstage-community-plugin-catalog-backend-module-pingidentity",
```

Comment:
do we want pingidentity in 1.3?

Reply:
Oh, maybe I shouldn't have included that, that was one of the merge conflicts